WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsStripeAcountReviewView.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  InPersonPaymentsStripeAcountReview.swift
+//  WooCommerce
+//
+//  Created by Cesar Tardaguila on 2021-08-10.
+//  Copyright © 2021 Automattic. All rights reserved.
+//
+
```

Comment:
```suggestion
```

IIRC, we don't do file headers. I wish I could make Xcode learn that 😅

Reply:
Argh, I hate when I forget to remove these. Sorry!